modin/pandas/dataframe.py

```diff
@@ -2416,6 +2413,27 @@ def _getitem(self, key):
         else:
             return self._getitem_column(key)
 
+    # persistance support
```

Comment:
Maybe this comment should be above `def _inflate_full` function, not `def _inflate_light`?

Reply:
this comment denotes the place where persistance support methods start... I should probably make it clearer.

Reply:
I tried to make it clearer by rewording the comments.